common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -754,82 +754,41 @@ public static int hashCodeAscii(byte[] bytes, int startPos, int length) {
      */
     public static int hashCodeAscii(CharSequence bytes) {
         int hash = HASH_CODE_ASCII_SEED;
-        final int remainingBytes = bytes.length() & 7;
+        final int length = bytes.length(), remainingBytes = length & 7;
         // Benchmarking shows that by just naively looping for inputs 8~31 bytes long we incur a relatively large
         // performance penalty (only achieve about 60% performance of loop which iterates over each char). So because
         // of this we take special provisions to unroll the looping for these conditions.
```

Comment:
@njhill is this comment still true ?

Reply:
Not sure, I can test it...

Reply:
would be nice 👍

Reply:
@normanmaurer interesting, it does appear to still be the case.

This PR as-is:
```
AsciiStringBenchmark.hashCodeBenchCharSequenceNew       8  thrpt    5   84938768.167 ±  3807502.233  ops/s
AsciiStringBenchmark.hashCodeBenchCharSequenceNew      10  thrpt    5   75864099.463 ±  3345939.549  ops/s
```

Without 8-31 bytes special case:
```
AsciiStringBenchmark.hashCodeBenchCharSequenceNew       8  thrpt    5   55068716.482 ±  8293068.816  ops/s
AsciiStringBenchmark.hashCodeBenchCharSequenceNew      10  thrpt    5   52594830.642 ±  4131819.596  ops/s
```

Reply:
@njhill thanks for testing :)